src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouteTab, RouteTabs } from "v2/Components/RouteTabs"
+import { NavigationTabs_partner } from "v2/__generated__/NavigationTabs_partner.graphql"
+
+interface NavigationTabsProps {
+  partner: NavigationTabs_partner
+}
+
+export class NavigationTabs extends React.Component<NavigationTabsProps> {
```

Comment:
I'm not sure if we have a style guide that mentions this anywhere but we should avoid class components in favor of functional components.

Reply:
Fixed